driver-core/src/main/java/com/datastax/driver/core/Configuration.java

```diff
@@ -52,7 +53,9 @@ public Configuration() {
              new SocketOptions(),
              new MetricsOptions(),
              new QueryOptions(),
-             NettyOptions.DEFAULT_INSTANCE);
+             NettyOptions.DEFAULT_INSTANCE,
+             new CodecRegistry()
```

Comment:
Actually there is no default instance, as a codec registry is mutable and evolves over time specially when UDTs and tuples come into play.